wrapper.py

```diff
@@ -16,13 +17,14 @@ def disable_reflection():
         ChatWrapper.reflection_enabled = False
 
     @staticmethod
-    def log_to_ndjson(params, response):
+    def log_to_ndjson(params, response, response_time):
         timestamp = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
         model = params.get("model", "")
         total_tokens = response["usage"]["total_tokens"]
 
         log_entry = {
             "timestamp": timestamp,
+            "response_time": round(response_time, 2),
```

Comment:
Added rounding to 2 decimal points, imo very little utility to be more granular than that measuring http api calls.